### ISSUE TITLE
Help needed with rancher installation

### ISSUE THREAD

**Issue by laralar** [rancher/rancher#48355]
Hello

I've deployed a standard RKE2 cluster using rancher.

I am trying to install rancher on that second cluster

```
helm install rancher rancher-latest/rancher --namespace cattle-system --set hostname=my-declared-host-etc-hosts --set ingress.tls.source=rancher --set replicas=3 --set bootstrapPassword=password
Error: INSTALLATION FAILED: 2 errors occurred:
	* Internal error occurred: failed calling webhook "validate.nginx.ingress.kubernetes.io": failed to call webhook: Post "https://rke2-ingress-nginx-controller-admission.kube-system.svc:443/networking/v1/ingresses?timeout=10s": context deadline exceeded
	* Internal error occurred: failed calling webhook "webhook.cert-manager.io": failed to call webhook: Post "https://cert-manager-webhook.cert-manager.svc:443/validate?timeout=30s": context deadline exce
```

I am having the same issue while trying to install nextcloud

```
helm install nextcloud nextcloud/nextcloud -f nextcloud.yaml
Error: INSTALLATION FAILED: 1 error occurred:
	* Internal error occurred: failed calling webhook "validate.nginx.ingress.kubernetes.io": failed to call webhook: Post "https://rke2-ingress-nginx-controller-admission.kube-system.svc:443/networking/v1/ingresses?timeout=10s": context deadline exceeded
```

Also.. if I deploy nextcloud in the original RK2 cluster, there is no issue, so there is a difference in configuration between the two RkE2 clusters
```
helm install nextcloud nextcloud/nextcloud -f nextcloud.yaml -n nextcloud --create-namespace
NAME: nextcloud
LAST DEPLOYED: Tue Dec 10 05:35:30 2024
NAMESPACE: nextcloud
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
1. Get the nextcloud URL by running:

  export POD_NAME=$(kubectl get pods --namespace nextcloud -l "app.kubernetes.io/name=nextcloud" -o jsonpath="{.items[0].metadata.name}")
  echo http://127.0.0.1:8080/
  kubectl port-forward --namespace nextcloud $POD_NAME 8080:80

2. Get your nextcloud login credentials by running:

  echo User:     admin
  echo Password: $(kubectl get secret --namespace nextcloud nextcloud -o jsonpath="{.data.nextcloud-password}" | base64 --decode)
root@kadmin:~# 
```

First RKE2 Cluster, where nextcloud deployment works is manually created, and rancher installed there manually.  It works fine.  I didn't want to install nextclud there, so I created another RKE2 cluster, but this time custom deployed from Rancher.  In that custom deployed cluster from rancher is where I cannot install neither rancher, nor nextcloud 

I am very new to K8s.
	
Any hint?
	
Thanks

**Comment by bpedersen2** [comment]
1) Why would you want more than 1 rancher installation?  rancher can manage  lots of downstream clusters.
2) did you disable  the ingress controller in the rke2 installation for the second cluster?

**Comment by laralar** [comment]
As I said I am very new to K8S, the error started with Nextcloud installation in a RKE2 rancher provisioned cluster that was failing setting up the ingress, then I rememebered in the rancher installation I did, I had to setup an ingress. so I tried installing rancher on this new cluster and it failed with the same error.

the first rancher installation of rancher was not that "elegant" let's say.  And I wanted a clean RKE2 installation and installing rancher on it, and let go of the first installation.  

I did not disable the ingress controller upon installation, the pods were there, just not responding.  But the fleet-agent was not active at all, stuck in initializing.   At the end it had to do with some underlying dns issue at my end.

I did a clean reinstall of rancher using the K3S CLI Quick start guide 
https://ranchermanager.docs.rancher.com/getting-started/quick-start-guides/deploy-rancher-manager/helm-cli

and worked like a charm. After this I did a second RKE2 installation (without rancher) and I was just able to install nextcloud on it without the ingress issues. I think it was DNS related, (fleet is active)
